src/nanoemoji/maximum_color.py

```diff
@@ -238,14 +294,62 @@ def _generate_colr_from_svg(nw: NinjaWriter, input_font: Path, font: ttLib.TTFon
     )
     nw.newline()
 
-    # stick our shiny new COLR table onto the input font and declare victory
+    # stick our shiny new COLR table onto the input font
+    output_file = Path(input_font.stem + ".both_vector_tables.ttf")
     nw.build(
-        input_font.name,
+        output_file,
         "copy_colr_from_svg2colr",
         "colr_from_svg.ttf",
     )
     nw.newline()
 
+    return output_file, picosvgs
+
+
+def _generate_cbdt(
+    nw: NinjaWriter,
+    input_font: Path,
+    font: ttLib.TTFont,
+    color_font: Path,
+    picosvg_files: List[Path],
+):
+    # generate bitmaps
+    bitmap_files = [rel_build(bitmap_dest(s)) for s in picosvg_files]
+    for picosvg, bitmap in zip(picosvg_files, bitmap_files):
+        nw.build(bitmap, "write_bitmap", picosvg)
+    nw.newline()
+
+    # make a glyphmap
+    nw.build(
+        _CBDT_GLYPHMAP,
+        "write_glyphmap_for_glyph_svgs",
+        picosvg_files + bitmap_files + [input_font],
+    )
+    nw.newline()
+
+    # make a config
+    nw.build(_CBDT_CONFIG, "write_cbdt_config", input_font)
+    nw.newline()
+
+    # generate a new font with CBDT glyphs that use the same names as the original
+    # hidden in the rule definition is that the both vector table font is an input
+    # so list it as implicit
```

Comment:
is that true that the ".both_vector_tables.ttf" font is hidden in the "write_cbdt_font" rule definition? I don't see that..
The "write_cbdt_font" simply calls write_font.py with the cbdt glyphmap and cbdt config to produce a cbdt.ttf. No mention of the ".both_vector_tables.ttf" font.

however it _is_ included in the `copy_cbdt` rule definition (which calls glue_together.py), maybe that's what you meant?

Reply:
Hm ... I think it might be off by one indeed.